src/main/java/com/github/maxopoly/WurstCivTools/effect/LeafShears.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * @author Aleksey Terzi
+ *
+ */
+
+package com.github.maxopoly.WurstCivTools.effect;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.ItemStack;
+
+import vg.civcraft.mc.citadel.Citadel;
+import vg.civcraft.mc.citadel.PlayerState;
+import vg.civcraft.mc.citadel.reinforcement.PlayerReinforcement;
+import vg.civcraft.mc.citadel.reinforcement.Reinforcement;
+
+import com.github.maxopoly.WurstCivTools.WurstCivTools;
+
+public class LeafShears extends WurstEffect {
+	private int clearCubeSize;
+
+	public LeafShears(int clearCubeSize) {
+		super();
+		
+		this.clearCubeSize = clearCubeSize;
+	}
+
+	@Override
+	public void handleBreak(Player p, BlockBreakEvent e) {
+		if(e.isCancelled()) {
+			return;
+		}
+		
+		final Player player = p;
+		
+		if(player == null) {
+			return;
+		}
+		
+		if(!isLeaf(e.getBlock())) {
+			return;
+		}
+		
+		e.setCancelled(true);
+		e.getBlock().setType(Material.AIR);
+		
+		//Following manipulations are needed to synchronize tool durability with client after cancel block break
+		final ItemStack handItem = player.getInventory().getItemInMainHand();
+
+		handItem.setDurability((short)(handItem.getDurability() + 1));
+		
+		final Location center = e.getBlock().getLocation();
+		
+		Bukkit.getScheduler().runTask(WurstCivTools.getPlugin(), new Runnable() {
+            public void run() {
+            	handItem.setDurability((short)(handItem.getDurability() - 1));
```

Comment:
Ok...but this is additional problems - if tool has Unbreakable enchantment - then I will need to introduce little more code to correctly process this.

Also if you want to force durability loss - we need to decided how tool will loose this durability.
Right now - if I'm using shears  - they will be broken almost immediately after I start clearing leaves.  

This will not allow to clear big amount of leaves we have in Eilon.

Remember experience with custom leaves in Devoted - internal regions of southern continent were almost unpopulated, because of people haven't possibility to clear forest (after fire spreading was disabled), but iron on Devoted was very and very cheap.

Please let me know how you think this should work.